packages/admin/admin/src/form/file/getFilesInfoText.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { PrettyBytes } from "../../helpers/PrettyBytes";
+import { FileSelectProps } from "./FileSelect";
+
+export const getFilesInfoText = (maxFiles: FileSelectProps["maxFiles"], maxFileSize: FileSelectProps["maxFileSize"]) => {
```

Comment:
Why not make this a React component?

Reply:
I just want a simple function to just get the text. 

When changing this to a react-component, I can't do a null-check anymore, so I would have to move `<FilesInfoText />` into the component as well, and then pass it's `slotProps`. Keeping it as a function makes the code more readable imo.